packages/neo-one-website/docs/2-advanced-guides/10-configuration-options

```diff
@@ -0,0 +1,41 @@
+---
+slug: config-options
+title: Configuration Options
+---
+NEO•ONE compiler configuration options
+
+Configure NEO•ONE for your project.
+
+## Config File
+
+An [initialized environment](https://neo-one.io/tutorial#Setup-for-the-Tutorial) will have a ` .onerc ` file within the project root.
+
+### Example NEO•ONE configuration file
+
+The `.onerc` configuration file could look something like this:
+```json
+{
+  "codegen": {
+    "language": "typescript",
+    "framework": "react",
+    "browser":  true
```

Comment:
Change this to false.  The default is false and they really shouldn't be using the browser option except for specific use cases.  So for people copy-pasting from here or something, we would want it false.

Reply:
updated: https://github.com/neo-one-suite/neo-one/pull/1177/files#diff-172ea3c052d9b136cba8549a61c6fd9dR21